docs/snippets/gettingStarted/webApi/simpleApi.ts

```diff
@@ -139,6 +141,40 @@ export const shoppingCartApi =
         return NoContent();
       }),
     );
+
+    // Get Shopping Cart
+    router.get(
```

Comment:
**FIX:** @thiagomini, for now I'd prefer to keep the sample outside of the `complete-api` region as the example in the getting started is (for now) focused on business logic and writing. I'm planning to introduce the read model side as the next step. Would you mind moving this endpoint outside the region?